.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  rspec:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { ruby: '2.3', rails: '4.2' }
+          - { ruby: '2.4', rails: '5.0' }
+          - { ruby: '2.5', rails: '5.1' }
+          - { ruby: '2.6', rails: '5.2' }
+          - { ruby: '2.6', rails: '6.0' }
```

Comment:
> ruby-2.7 fails with undefined method 'new' for BigDecimal:Class: ojab/active_data/runs/4517620672?check_suite_focus=true
:P  

There are related changes in https://github.com/pyromaniac/active_data/pull/77, I'll try to apply the same change here and see if nothing will break. Don't want to create a conflict, OTOH it would be minimal if any.

Reply:
Yep, change from #77 works even for 2.3, test run is green https://github.com/ojab/active_data/runs/4517810478?check_suite_focus=true, pushed.